CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - Improve API response when an error occurs by including information about the cause ([#186])
 - Add parameter to get API response without svg string in the result ([#137])
 - Extend time range for Currentness indicator ([#178])
+- Rewrite the Mapping Saturation indicator to use statistical models from the R language ([#209] [#170])
```

Comment:
- [ ] this change isn't in 0.7.0
- [ ] please add a line to “How to upgrade”
- [ ] `[#209]` missing
- [ ] consistent would be:
```suggestion
- Rewrite the Mapping Saturation indicator to use statistical models from the R language ([#170], [#209])
```